### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by jarno-k**
Improved documentation such that it can automatically generate pages from gritulator docstrings and simulation examples. In addition fixed some errors and warnings.

### CODE REVIEW

File: docs/source/model/system.rst [hunk 1]
```diff
@@ -1,21 +1,38 @@
...
+Grid-converter System
+=====================
 
-Machine drives are sampled-data systems, consisting of continuous-time systems and discrete-time systems as well as the interfaces between them [#Fra1997]_, [#Bus2015]_. The figure below shows a generic example system. The same architecture is used in *motulator*.
+Grid-converter systems are sampled-data systems, consisting of continuous-time 
+systems and discrete-time systems as well as the interfaces between them 
+[#Fra1997]_, [#Bus2015]_. The figure below shows a generic example system. 
+The same architecture is used in *gritulator*.
 
 .. figure:: figs/system.svg
```
Comment (rayanmour) [inline docs/source/model/system.rst:9]: Figure has been updated

File: docs/source/model/filter_and_grid.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+   :label: zero_sequence
+
+.. Even though the zero-sequence voltage exists at the ouput of typical 
+.. converters (see :doc:`/model/converters`), there is no path for the 
+.. zero-sequence current to flow in three-phase three-wire grid-converter systems, 
+.. i.e., :math:`i_\mathrm{c0} = 0`.
+
+The space vector transformation in :eq:`space_vector` is implemented in the 
+function :func:`gritulator.abc2complex` and its inverse transformation in the 
+function :func:`gritulator.complex2abc`.
```
Comment (rayanmour) [inline docs/source/model/filter_and_grid.rst:37]: Maybe we could mention the package ":mod:`gritulator.model.grid_filter'" before jumping in the L and LCL filter sections?

File: docs/source/model/system.rst [hunk 3]
```diff
@@ -1,21 +1,38 @@
...
-
-Digital control systems typically have a computational delay of one sampling period, :math:`N=1`. The PWM block shown in the figure models the carrier comparison, see more details in :doc:`converters`. The carrier comparison is implemented in the class :class:`motulator.model.CarrierComparison`. If the switching ripple is not of interest in simulations, the carrier comparison can be replaced with zero-order hold (ZOH).
+   Block diagram of a sampled-data system. Discrete signals and systems are 
+   shown in blue, and continuous signals and systems are shown in red. 
+
+The continuous-time system (named `mdl` in :doc:`/auto_examples/index`) is the 
+model of a physical grid-converter system, typically consisting of a power 
+converter, grid, and a filter, such as an LCL filter, between the converter and 
+grid. The continuous-time system may have external inputs, such as a power fed
```
Comment (rayanmour) [inline docs/source/model/system.rst]: the grid?

File: docs/source/model/system.rst [hunk 4]
```diff
@@ -1,21 +1,38 @@
...
+   shown in blue, and continuous signals and systems are shown in red. 
+
+The continuous-time system (named `mdl` in :doc:`/auto_examples/index`) is the 
+model of a physical grid-converter system, typically consisting of a power 
+converter, grid, and a filter, such as an LCL filter, between the converter and 
+grid. The continuous-time system may have external inputs, such as a power fed 
+to DC-bus of the converter.
+
+The discrete-time controller (named `ctrl`) contains control algorithms, such as 
+the current control and DC-bus voltage control. The reference signals could be,
```
Comment (rayanmour) [inline docs/source/model/system.rst]: such as current control

File: docs/source/model/system.rst (hunk 1 above)
Comment (jarno-k) [reply docs/source/model/system.rst:9]: Yes.

File: docs/source/model/filter_and_grid.rst (hunk 2 above)
Comment (jarno-k) [reply docs/source/model/filter_and_grid.rst:37]: Those different grid filters can be referred when augmenting the documentation.

File: docs/source/model/system.rst (hunk 3 above)
Comment (jarno-k) [reply docs/source/model/system.rst]: revised

File: docs/source/model/system.rst (hunk 4 above)
Comment (jarno-k) [reply docs/source/model/system.rst]: revised

**Pull request merged**